services/deps-rs/deps-rs-crate.tester.js

```diff
@@ -0,0 +1,21 @@
+import Joi from 'joi'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('dependencies (valid crate, latest version)')
+  .get('/syn/latest.json')
+  .expectBadge({
+    label: 'dependencies',
+    message: Joi.string(),
```

Comment:
Just testing for "string" here will match any string, including errors. So this test will never fail if something changes upstream, which kind of negates the point of the live integration test. Not found, invalid response.. they'll all match here.

I think what would make sense here would be to either make a Joi validator which accepts the strings (or string patterns) we know about from the upstream service and then use that to validate both the input and also use the same validator in the service tests.